### ISSUE TITLE
Make Presentation for Demo 

### ISSUE THREAD

**Issue by chrisfernandes18** [uchicago-cs/chiventure#307]
We need to make a presentation for the final presentation to show how we came up with our design (showing how we decided between protobuf and messagepack), how we implemented our project, and a demonstration of how we save and load our game.

**Comment by jarvislam1999** [comment]
I wrote a bunch of contents for save and load, and also added a few slides to discuss game.proto and future directions. Could you guys give it a check?

**Comment by schin99** [comment]
Started a document to write a script! Anything you'd like to say that's not on the slide, feel free to include it in the document.

**Comment by chrisfernandes18** [comment]
Wrote a comparison page  between the external libraries we wanted to use whether we wanted to use protobuf or message pack. Also helped create the "demo" for the powerpoint that Sheryl is going to present on.

**Comment by chrisfernandes18** [comment]
Well our presentation is complete and ready for tomorrow. We integrated the feedback that Borja and Mark gave us and have the presentation done. We made our introduction clearer for what our group does and then how we decided to tackle our issue. Then how we decided to checkpoint the game and the challenges we faced in that. The final part was what we wanted to checkpoint from the game and then what we might want to implement in the future for checkpointing a game. It's been a dope time working with you guys. We've got the presentation and then hope everyone has a dope summer!
